project_event/wizard/project_event_wizard.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding:utf-8 -*-
+# @ 2018 Savoir-failre Linux
+# License LGPL-3.0 or Later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models, _
+
+
+class ProjectEventWizard(models.TransientModel):
+    _name = "project.event.wizard"
+    _description = "Wizard Event Creation from Template"
+
+    template_id = fields.Many2one(
+        'event.template',
+        string='Event Template',
+    )
+    name = fields.Char(
+        string='Event Title',
+    )
+    event_resp_id = fields.Many2one(
+        'res.partner',
+        string='Responsible',
+    )
+    event_notes = fields.Text(
+        string='Notes',
+    )
+    activity_line_ids = fields.One2many(
+        'activity.template.line.wiz',
+        'event_wiz_id',
+        string='Activities to Edit'
+    )
+    task_line_ids = fields.One2many(
+        'task.template.line.wiz',
+        'event_wiz_id',
+        string='Tasks to Edit'
+    )
+
+    @api.multi
+    def add_activities(self):
+        self.ensure_one()
+        template = self.env['event.template'].browse(self.template_id.id)
+        view = self.env.ref(
+            'project_event.view_project_activity_wizard')
```

Comment:
@rimbendhaou: Refer the previous comment "view_project_activity_wizard" naming convention is not match with other files in the project event model